content/chat/rooms/messages.textile

```diff
@@ -126,3 +140,118 @@ const MyComponent = () => {
   );
 };
 ```
+
+h2(#delete). Delete a message
+
+blang[javascript].
+  Use the "@messages.delete()@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/chat_js.Messages.html#delete method to delete a message in a chat room. All users that are "subscribed":#subscribe to messages on that room will receive the deletion:
+
+blang[react].
+  Use the "@deleteMessage()@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/chat_react.UseMessagesResponse.html#deleteMessage method available from the response of the @useMessages@ hook to delete a message from the room:
+
+```[javascript]
+import { Message } from '@ably/chat';
+const messageToDelete: Message = // Some Message object
```

Comment:
I think rather than putting a comment `// some message object`, it would probably be better just to do

`const messageToDelete: Message`

I think it looks cleaner that way